### PR TITLE
wip: not to merge

### PR DESCRIPTION
**Pull request by ShragaUser**
Readme

### CODE REVIEW

**Comment by ShragaUser** [comment]
to run with new setup you need to run
```bash
pnpm install
export N8N_CUSTOM_EXTENSIONS=<N8N_REPO_LOCATION_ON_LOCAL_COMPUTER>/packages/n8n-nodes-honeybook/dist 
pnpm run dev # or pnpm run start if you already ran build
```

at build time we will build the package directly to ~/.n8n/nodes directory where it is loaded automatically as a community node, we do not have to "enable community nodes" to enable this.

File: packages/n8n-nodes-honeybook/package.json [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+    "format": "prettier --write . --ignore-path ../../.prettierignore",
+    "lint": "eslint . --quiet && node ./scripts/validate-load-options-methods.js",
+    "lintfix": "eslint . --fix",
+    "watch": "tsc-watch -p tsconfig.build.json --onCompilationComplete \"tsc-alias -p tsconfig.build.json\" --onSuccess \"pnpm n8n-generate-ui-types\"",
+    "test": "jest"
+  },
+  "files": [
+    "dist"
+  ],
+  "n8n": {
```
Comment (ShragaUser) [inline packages/n8n-nodes-honeybook/package.json:22]: only need to update this to export new nodes and credentials - only things here will actually be exported - note that if something is here and is not defined in code, build will fail.

**Comment by Joffcom** [comment]
Hey @ShragaUser,

Thanks for the contribution, We have now created a Community Node Repository which is our preferred method for new node submissions.

Once a node has been published to the Community Node Repository users will be able to install the node from their n8n interface and will be able to use your node quicker, You will also be able to update the node as needed without needing to wait for the review process.

In the future we may reach out to you about including your node in our main release, But for now I am going to close this PR.

You can find more information in the links below, If you have any questions about this or need any help let us know.

Building Community Nodes: https://docs.n8n.io/integrations/community-nodes/build-community-nodes/
Submitting Community Nodes: https://docs.n8n.io/integrations/creating-nodes/deploy/submit-community-nodes/
Published Nodes: https://www.npmjs.com/search?q=keywords:n8n-community-node-package

**Comment by ShragaUser** [comment]
Sorry that's on me - was mistakenly opened  🙏

**Pull request closed without merging**